services/gateway/sources/mappings/account.js

```diff
@@ -55,4 +67,9 @@ module.exports = {
 		incoming: 'incomingTxsCount,string',
 		outgoing: 'outgoingTxsCount,string',
 	},
+	unlocking: ['unlocking', {
+		amount: '=,string',
+		height: 'unvoteHeight,number',
```

Comment:
As per the `SDKv4 Impact Analysis on Lisk Service` document, `height` must be an object with the properties `start` and `end`. Kindly make necessary changes to reflect the same.

Perhaps it'll result in something similar to as suggested below. Kindly verify and modify as necessary:
```suggestion
		height: {
			start: 'unvoteHeight.start,string',
			end: 'unvoteHeight.end,string',
		}
```

Reply:
@MichalTuleja @sameersubudhi : Got clarification on this from @UsamaHameed. Structure `UI` need from lisk-service for unlocking is:
```unlocking: [{ 
        amount: '1000000000',
        unvoteHeight: 280000, 
        delegateAddress: '1L' 
}]```
In frontend, they already take care of this.